Makefile

```diff
@@ -44,15 +44,11 @@ PULL_BASE_SHA ?= master
 
 .PHONY: lint
 lint:
-	hack/golangci-lint.sh run --new-from-rev=${PULL_BASE_SHA}
+	hack/golangci-lint.sh run --new-from-rev=${PULL_BASE_SHA} ${EXTRA_ARGS}
 
-# While https://github.com/golangci/golangci-lint/issues/1779 is not fixed,
-# we need to run the fix separately from the lint command.
-# GolangCI-Lint will not actually run the fixer for us.
-# In the future we can remove this and have the linter auto-fix.
 .PHONY: lint-fix
 lint-fix:
-	hack/lint-fix.sh
+	EXTRA_ARGS=--fix make lint
```

Comment:
Nit: I generally don't like calling `make` within the Makefile itself. Totally a personal preference, but as a reader that knows the basics of `make` and Makefiles it always throws me off a bit to see it. Usually I go more for something like:
```make
.PHONY: lint-fix
lint-fix: EXTRA_ARGS=--fix
lint-fix: lint
```